apps/ui/src/api/useCreateApplication.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2025 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import { useMutation } from '@tanstack/react-query';
+import { notification } from 'antd';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router';
+
+import { mockUserID } from '@/components/mock/applicationMockData';
+import { fetch } from '@/global/FetchClient';
+import { ServerError } from '@/global/types';
+import { Application } from '@pcgl-daco/data-model';
+
+const useCreateApplication = () => {
+	const navigation = useNavigate();
+	const { t: translate } = useTranslation();
+
+	return useMutation<Application, ServerError>({
```

Comment:
abstracting `useMutation` into a function and then renaming the variable, you may as well return the variable the way you want it in the first place, it's more descriptive than just `useMutation`. `const { createApplication} = useCreateApplication()`